docs/installation.md

```diff
@@ -1,3 +1,13 @@
+# Quick install
+
+Run below command to install latest version of aeternity node.
+
+```bash
+wget https://raw.githubusercontent.com/aeternity/epoch/v1.2.0/scripts/install.sh && chmod +x install.sh && ./install.sh 1.2.0
```

Comment:
Mentioned for smoke testing in release notes - @dincho please review.
https://github.com/aeternity/epoch/wiki/Release-process/_compare/9e4071527559570bb5d852e01f0c9ad0c2a04072...a779f4fa113135666621ec1d015b2464e9eeef33